webhooks-extension/config/latest/gcr-tekton-webhooks-extension.yaml

```diff
@@ -8,52 +8,175 @@ metadata:
   namespace: tekton-pipelines
 ---
 # ------------------- Extension Role & Role Binding ------------------- #
```

Comment:
nit: please remove this line for consistency with the `webhooks-extension/config/extension-deployment.yaml`

Reply:
Hi @ncskier powers granted by a ClusterRole apply across all namespaces. Powers granted by a Role apply only in the one namespace. So this change greatly the ability of the `tekton-webhooks-extension` to operate on resources in other namespaces to the absolute minimum. 

Please note that I am rationalising all our yaml to single files under https://github.com/tektoncd/experimental/issues/394 (which thank you for raising) so I'm not too bothered about consistency between the various deployments in these last few days before they're properly fixed under 394.